pkg/retester/retester.go

```diff
@@ -49,7 +57,7 @@ func (b *backoffCache) loadFromDisk() error {
 	if err != nil {
 		return fmt.Errorf("failed to read file %s: %w", b.file, err)
 	}
-	cache := map[string]*PullRequest{}
+	cache := map[string]*pullRequest{}
```

Comment:
Could you please add unit tests for `loadFromDisk` and `saveToDisk`?
I want to make sure `pullRequest` is still working for them.

Reply:
Tests added in last commit.